doc/user/content/sql/create-sink.md

```diff
@@ -163,6 +165,22 @@ You can find the topic name for each Kafka sink by querying `mz_kafka_sinks`.
 
 {{% kafka-sink-drop  %}}
 
+#### Enabling topic reuse after restart
+
+{{< version-added v0.8.2 />}}
+
+This feature is only available in experimental mode. By default, Materialize creates new, distinct topics for sinks after each restart. To enable the reuse of the existing topic instead, when you create a sink, you must:
+
+* Enable the `exactly_once` switch.
+* Specify a consistency topic to store the information Materialize will use to identify where synching left off. For more information, see [Consistency metadata](#consistency-metadata).
+
+To synchronize this history, Materialize must be able to reconstruct the history of the sinked object and all objects on which it is dependent. For Kafka, this means that the sinked topic should be set to infinite retention. Additionally, the name of the consistency_topic must be unique across all your sinks (not only the sink for which it tracks consistency data).
+
+Compaction may be set for the Kafka sink topic. We have not tested this with compaction set on the consistency topic; time-based compaction is *not* supported, but key-value compaction may work.
```

Comment:
We shouldn't have this sentence in the docs and instead should wait for engineering to test what settings do and don't work and then make a definite claim about those. @aljoscha have you tested exactly-once sinks with compaction on the sink / consistency topic?

Reply:
I think we can strike it. I haven't tested it, but for the sink to work, only the latest record needs to be retained. Everything else is ignored by the sinks, so it should just work. (famous last words, I know... :sweat_smile:)

Reply:
right but thats only on the consistency topic not the data topic right? maybe it will all work but I would feel more comfortable after testing (which doesn't have to block the docs pr)